crates/spin-wasip3-http/src/lib.rs

```diff
@@ -5,9 +5,14 @@
 #[doc(hidden)]
 pub use wasip3;
 
+pub use wasip3::{
+    http_compat::{IncomingMessage, Request, Response},
+    wit_bindgen::{self, spawn},
```

Comment:
I guess the train has already left the station, but re-exporting `wasip3` which itself includes all of `wit_bindgen` makes backwards compat very difficult. We're tying ourselves to the exact public API of two external crates. Maybe that's unavoidable...

Reply:
I think `wit-bindgen` is effectively "`std`-like" for async Rust guests. `wasip3` is a little different but we have to at least _secretly_ re-export it to make `#[http_service]` work.